v1.1: Add new filter rule `_geoBoundingBox` (#2216)
meilisearch/documentation: learn/advanced/filtering_and_faceted_search.md

release_date > 1577884550 AND (NOT director = "Tim Burton" AND director EXISTS)
```

### Filtering by geographic location

> If your documents contain `_geo` data, you can filter results according to their geographic position using one of our built-in geosearch filter rules:

Reviewer: ```suggestion
If your documents contain `_geo` data, you can filter results according to their geographic position using our built-in geosearch filter rules `_geoRadius` and `_geoBoundingBox` :
```

Author: I actually did this on purpose. It doesn't seem unlikely the devs will create new geosearch filters (for instance, `_geoPolygon` that lets users search in shapes more complex than rectangles and circles), so I'm trying to keep it open in pages like this one, where geosearch is not the main subject.
